Sample_v3/Samples/SwiftUISimpleChat/ChatView.swift

```diff
@@ -18,23 +18,17 @@ struct ChatView: View {
     @State var actionSheetTrigger: Bool = false
     /// Message being edited.
     @State var editingMessage: ChatMessage?
-    
-    /// User action.
-    @State var userActionTrigger: Bool = false
-    @State var userAction: ((String) -> Void)?
-    
+    /// Member list trigger.
+    @State private var isShowingMemberList = false
+
     var body: some View {
         VStack {
             self.messageList().layoutPriority(1)
             self.composerView()
+            NavigationLink(destination: memberList, isActive: $isShowingMemberList) { EmptyView() }
```

Comment:
I'm that familiar with SwiftUI but the `EmptyView()` looks like a leftover here :) All good if it's intentional.

Reply:
Me neither but seems it is required. We are using `public init(destination: Destination, isActive: Binding<Bool>, @ViewBuilder label: () -> Label)` and the `label` closure is required.

Do we want to extend the `NavigationLink` to have  `public init(destination: Destination, isActive: Binding<Bool>)` that will hide this label details?

Reply:
All good, please ignore my comment.